README.md

```diff
@@ -350,6 +350,7 @@ The following settings can be optionally set to customize the node labels and ta
 The following settings are set for you automatically by [pluto](sources/api/) based on runtime instance information, but you can override them if you know what you're doing!
 * `settings.kubernetes.max-pods`: The maximum number of pods that can be scheduled on this node (limited by number of available IPv4 addresses)
 * `settings.kubernetes.cluster-dns-ip`: The CIDR block of the primary network interface.
+* `settings.kubernetes.cluster-domain`: The cluster domain is set to `cluster.local` by default.
```

Comment:
We currently have three subsections related to Kubernetes settings:
* Required settings  (this isn't required)
* Optional settings, but specifically narrowed down to node labels/taints
* This section, which is for for settings automatically set by `pluto` based on runtime information  (this isn't discovered at runtime)

I think instead of being here, we should make another section about optional settings, above the pluto section, since most people won't override the settings set by `pluto`. Perhaps like this:
```
The following settings are optional and allow you to further configure your cluster:
```

I also think it would be helpful to have a description of this setting before its default value, perhaps like this:
```
* `settings.kubernetes.cluster-domain`: The DNS domain for this cluster, allowing all Kubernetes-run containers to search this domain before the host's search domains.  Defaults to `cluster.local`.
```

Reply:
Yes. Makes sense.